config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-ec2.yaml

```diff
@@ -15,6 +15,10 @@ periodics:
     repo: kubernetes
```

Comment:
So we have presubmits in this location  https://github.com/kubernetes/test-infra/tree/c40ed5aa9f1e14d94a4a47aa8e4e61fdd1dc94f4/config/jobs/kubernetes/kops

and periodics in this location? Should we have them in the same place under config/jobs/kubernetes/kops?

Reply:
yep, we should